site/apache.md

```diff
@@ -0,0 +1,460 @@
+
+      - watch
+
+  - apiGroups:
+
+      - extensions
+
+    resources:
+
+      - ingresses
+
+    verbs:
+
+      - get
+
+      - list
+
+      - watch
+
+---
+
+kind: ClusterRoleBinding
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+
+metadata:
+
+  name: domain1-apache-webtier
+
+  labels:
+
+    weblogic.domainUID: domain1
+
+    weblogic.domainName: base_domain
+
+roleRef:
+
+  apiGroup: rbac.authorization.k8s.io
+
+  kind: ClusterRole
+
+  name: domain1-apache-webtier
+
+subjects:
+
+- kind: ServiceAccount
+
+  name: domain1-apache-webtier
+
+  namespace: default
+
+```
+
+
+Here are examples of the Kubernetes artifacts created by the WebLogic Operator:
+
+
+```
+
+NAME                            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+
+bash-4.2$ kubectl get all |grep apache
+
+deploy/domain1-apache-webtier   1         1         1            1           2h
+
+
+rs/domain1-apache-webtier-7b8b789797   1         1         1         2h
+
+
+deploy/domain1-apache-webtier   1         1         1            1           2h
+
+
+rs/domain1-apache-webtier-7b8b789797   1         1         1         2h
+
+
+
+po/domain1-apache-webtier-7b8b789797-lm45z   1/1       Running   0          2h
+
+
+svc/domain1-apache-webtier                      NodePort    10.111.114.67    <none>        80:30305/TCP      2h
+
+
+
+bash-4.2$ kubectl get clusterroles |grep apache
+
+domain1-apache-webtier                                                 2h
+
+
+
+bash-4.2$ kubectl get clusterrolebindings |grep apache
+
+domain1-apache-webtier                                    2h
+
+```
+
+
+### Use your own plugin WL module configuration
+
+
+Optionally you can fine turn the behavior of the Apache plugin by providing your own Apache plugin configuration for `create-weblogic-domain.sh` script to use. You put your custom_mod_wl_apache.conf file in a local directory, say `/scratch/apache/config` , and specify this location in the `create-weblogic-domain-inputs.yaml` file as follows.
+
+```
+
+# Docker volume path for APACHE
+
+# By default, the VolumePath is empty, which will cause the volume mount be disabled
+
+loadBalancerVolumePath: /scratch/apache/config
+
+```
+
+Once the loadBalancerVolumePath is specified, the `create-weblogic-domain.sh` script will use the custom_mod_wl_apache.config file in `/scratch/apache/config` directory to replace what is in the Docker image.
+
+The generated yaml files will look similar except that the lines that start with "#" will be un-commented like the following.
+
+```
+
+      volumes:
+
+      - name: "domain1-apache-webtier"
+
+        hostPath:
+
+          path: /scratch/apache/config 
+
+      containers:
+
+      - name: domain1-apache-webtier
+
+        image: store/oracle/apache:12.2.1.3
+
+        imagePullPolicy: Never
+
+        volumeMounts:
+
+        - name: "domain1-apache-webtier"
+
+          mountPath: "/config"
+
+```
+
+
+
+## When use Apache load balancer with a manually created WebLogic Domain
```

Comment:
Should this be "When to use the Apache..."

Reply:
I am changing it to "Use the Apache...."